src/queue-list/laboratory-tabs.component.tsx

```diff
@@ -48,18 +48,26 @@ const LaboratoryQueueTabs: React.FC = () => {
             <Tab>{t("referredTests", "Referred tests")}</Tab>
             <Tab>{t("reviewList", "Review")}</Tab>
             <Tab>{t("approveList", "Approved")}</Tab>
-            {extensions.map((extension, index) => (
-              <Tab
-                key={index}
-                className={styles.tab}
-                id={`${extension.meta.title || index}-tab`}
-              >
-                {t(extension.meta.title, {
-                  ns: extension.moduleName,
-                  defaultValue: extension.meta.title,
-                })}
-              </Tab>
-            ))}
+            {tabExtensions
```

Comment:
Please add the check here as well. Refer to previous comment

Reply:
hello @pirupius i added a check that makes sure that the` title` and `name` values of the `extension.meta `object are truthy before rendering the Tab which is automatically mapped to the corresponding TabPanel .
Kindly see the updated screenshot